ClosedXML/Utils/OpenXmlHelper.cs

```diff
@@ -13,5 +18,42 @@ public static bool GetBooleanValueAsBool(BooleanValue value, bool defaultValue)
         {
             return (value?.HasValue ?? false) ? value.Value : defaultValue;
         }
+
+        /// <summary>
+        /// Convert color in ClosedXML representation to specified OpenXML type.
+        /// </summary>
+        /// <typeparam name="T">OpenXML color type to convert the argument to.</typeparam>
+        /// <param name="xlColor">Color in ClosedXML format.</param>
+        /// <param name="isDifferential">Flag specifiying that the color should be saved in
+        /// differential format (affects the transparent color processing).</param>
+        /// <returns>The original color in OpenXXL format.</returns>
+        public static T ToOpenXmlColor<T>(XLColor xlColor, bool isDifferential = false)
+            where T : DocumentFormat.OpenXml.Spreadsheet.ColorType, new()
+        {
+            var openXmlColor = new T();
```

Comment:
`new T()` is quite slow and there's another pattern that is preferred. See https://blogs.msdn.microsoft.com/seteplia/2017/02/01/dissecting-the-new-constraint-in-c-a-perfect-example-of-a-leaky-abstraction/

Reply:
Interesting, thanks. Never tested the performance of this approach.